components/association-section/index.tsx

```diff
@@ -17,18 +17,52 @@ const AssociationSection: React.FC<{
       idAssociation = '',
       nomComplet = '',
       objet = '',
-      adresse = '',
+      adresseGestion = '',
+      adresseSiege = '',
+      libelleFamille = '',
+      formeJuridique = '',
+      datePublicationJournalOfficiel = '',
+      dateCreation = '',
+      dateDissolution,
+      telephone,
+      mail,
+      siteWeb,
+      agrement = [],
+      eligibiliteCEC,
     },
   } = uniteLegale;
 
   const data = [
     ['N° RNA (identifiant d’association)', formatIntFr(idAssociation)],
     ['Nom', nomComplet],
+    ['Famille', libelleFamille],
     ['Objet', objet],
-    ['Adresse', adresse],
+    ['Forme juridique', formeJuridique],
+    [
+      'Date de publication au journal officiel',
+      formatDate(datePublicationJournalOfficiel),
+    ],
+    ['Date de création', formatDate(dateCreation)],
+    ...(dateDissolution && dateDissolution > dateCreation
+      ? [['Date dissolution', formatDate(dateDissolution)]]
+      : []),
+    ['Adresse du siège', adresseSiege],
+    ['Adresse de gestion', adresseGestion],
+    ['Téléphone', telephone],
+    ['Email', mail],
+    ['Site web', siteWeb],
+    [
+      'Agrement admininstratif',
+      <ul>
+        {agrement.map((agr) => {
+          return <li>{agr.attributeur}</li>;
+        })}
```

Comment:
As in mockup -> would add date and and maybe agrement label

See : https://www.data-asso.fr/annuaire/association/45037469900129